traffic_router/core/src/main/java/com/comcast/cdn/traffic_control/traffic_router/core/loc/RegionalGeo.java

```diff
@@ -219,6 +251,11 @@ private static RegionalGeo parseConfigJson(final JsonNode json) {
                     LOGGER.error("RegionalGeo ERR: geoLocation empty");
                     return null;
                 }
+                // coordinate range
+                final List<RegionalGeoCoordinateRange> coordinateRanges = parseLocationJsonCoordinateRange(locationJson);
+                if (coordinateRanges != null) {
+                    regionalGeo.setCoordinateRanges(coordinateRanges);
```

Comment:
why do we set this here? doesnt look like we ever use the class level variable so maybe we can get rid of it and the getter/setter and this if statement?

Reply:
You're correct, I removed it, along with the if statement, and the getters and setters.